frontend/components/doctor/patient-info-modal-swiper.tsx

```diff
@@ -31,9 +32,27 @@ const PatientInfoModalSwiper = ({ patient, onMutate }: AppProps) => {
             body: JSON.stringify({
                 statusId: statusIdGiven,
             }),
-        });
-
-        onMutate();
+        })
+            .then(() => {
+                toast({
+                    title: "Success",
+                    description: "This status has been set as reviewed",
+                    status: "success",
+                    duration: 3000,
+                    isClosable: true,
+                });
+                onMutate();
+            })
+            .catch(err => {
+                console.log(err);
```

Comment:
should we really log the error? isn't a modal enough?

Reply:
its to replicate the other behaviors in the modal , but it can be removed